batcher/aligned-batcher/src/lib.rs

```diff
@@ -888,7 +885,14 @@ impl Batcher {
             .collect();
 
         let batch_merkle_tree: MerkleTree<VerificationCommitmentBatch> =
-            MerkleTree::build(&batch_data_comm);
+            match MerkleTree::build(&batch_data_comm) {
+                Some(batch_merkle_tree) => batch_merkle_tree,
+                None => {
+                    return Err(BatcherError::TaskCreationError(
+                        "Failed to Build Merkle Tree: Empty Batch".to_string(),
+                    ))
+                }
+            };
```

Comment:
Nit: this could be more brief by using `ok_or_else`:
```rust
        let batch_merkle_tree: MerkleTree<VerificationCommitmentBatch> =
            MerkleTree::build(&batch_data_comm)
                .ok_or_else(|| Error::TaskCreationError("Failed to Build Merkle Tree: Empty Batch".to_string()))?;
```

Reply:
Thanks @Oppen